recipes/pyfolio/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyfolio" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "e30ee8c1932b45c4ce066cc78aee957d939e02a9427e797e2f7cd582cb0938c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - ipython
+    - matplotlib
+    - numpy
+    - pandas
+    - pytz
+    - scipy
+    - scikit-learn
+    - seaborn
+    - pandas-datareader
+    - empyrical
+    - pymc3
+    - theano
+
+test:
+  imports:
+    - pyfolio
+    - pyfolio.tests
+
+about:
+  home: https://quantopian.github.io/pyfolio/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Could you please ask upstream to add the license to the `MANIFEST.in` for the future?

Reply:
Thanks @alen12345. :)

Reply:
https://github.com/quantopian/pyfolio/pull/518

Issued a pull request to include the LICENSE in a future version; when it is added, I will update the recipe!